### DISCUSSION TITLE
Collecting User Name and Email Before Assisting in Flowise Chatbot Implementation

### DISCUSSION THREAD

**Discussion by Prakashram27** [FlowiseAI/Flowise#2781] [Q&A]
I'm trying to implement a chatbot using Flowise and have started working on a conversational retrieval QA chain. However, there's an issue: before assisting, the bot needs to obtain the user's name and email. I'm unsure how to collect the user's name and email using Flowise AI.

**Comment by x-du** [comment]
In Configuration ->Leads.   Enable lead generation.  

![image](https://github.com/user-attachments/assets/dd148ad4-ddf6-4317-af6a-67f267994228)

**Reply by gerald-CL** [reply]
Hello, do you know if it's possible to change the names of the requested fields? It's mainly because we need to use them in other languages.
